package.json

```diff
@@ -142,8 +144,10 @@
     "lodash.includes": "^4.3.0",
     "lodash.isobject": "^3.0.2",
     "lodash.keys": "^4.0.8",
+    "lodash.mapvalues": "^4.6.0",
```

Comment:
I had a hard time getting types right for my own `mapValues` implementation so used the one from lodash to save time.

Reply:
Yeah, seems reasonable to me 👍